libraries/AP_HAL/board/linux.h

```diff
@@ -175,7 +175,7 @@
     #define HAL_PROBE_EXTERNAL_I2C_COMPASSES
     #define HAL_OPTFLOW_PX4FLOW_I2C_BUS 2
     #define HAL_RANGEFINDER_LIGHTWARE_I2C_BUS 2
-    #define HAL_WITH_UAVCAN 1
```

Comment:
Yes, definitely, Testing is one thing which a huge bottleneck for getting this PR through, if we can find users who can help with this, that would great. Linux driver is tested under SITL to a large extent as both are exactly the same, so that certainly increases trust a bit more. The CANTester library is created to ease the process of testing, so basically users don't even need to have CAN accessories to help test this.